SecurityMetadata/Xslt/Security-Metadata-SQL-Generator.xslt

```diff
@@ -108,17 +108,19 @@ END
 
             SET @claimId = SCOPE_IDENTITY()
         END
+    <xsl:if test="@relocate = 'true'">
```

Comment:
I found the requirement to match the full hierarchy of the resource claims to be particularly challenging and accidental relocations were occurring and were too subtle too detect. So I've added support for a `relocate` attribute to the `<Claim>` element to only process a change in the `ParentResourceClaimId` if it is explicitly requested by the XML metadata.